spec/20-http_request_header_format.md

```diff
@@ -80,22 +81,25 @@ The dash (`-`) character is used as a delimiter between fields.
 #### version
 
 ``` abnf
-version         = 2HEXDIGLC   ; this document assumes version 00. Version ff is forbidden
+version         = 2HEXDIGLC   ; this document assumes version 01. Version ff is forbidden
 ```
 
 The value is US-ASCII encoded (which is UTF-8 compliant).
 
-Version (`version`) is 1 byte representing an 8-bit unsigned integer. Version `ff` is invalid. The current specification assumes the `version` is set to `00`.
+Version (`version`) is 1 byte representing an 8-bit unsigned integer. Version `ff` is invalid. The current specification assumes the `version` is set to `01`.
 
 #### version-format
 
-The following `version-format` definition is used for version `00`.
+The following `version-format` definition is used for version `01`.
 
 ``` abnf
-version-format   = trace-id "-" parent-id "-" trace-flags
-trace-id         = 32HEXDIGLC  ; 16 bytes array identifier. All zeroes forbidden
-parent-id        = 16HEXDIGLC  ; 8 bytes array identifier. All zeroes forbidden
-trace-flags      = 2HEXDIGLC   ; 8 bit flags. Currently, only one bit is used. See below for details
+version-format         = trace-id "-" parent-id "-" trace-flags [ "-" sampling-constant ]
+trace-id               = 32HEXDIGLC  ; 16 bytes array identifier. All zeroes forbidden
+parent-id              = 16HEXDIGLC  ; 8 bytes array identifier. All zeroes forbidden
+trace-flags            = 2HEXDIGLC   ; 8 bit flags. Currently, only one bit is used. See below for details
+sampling-constant      = sampling-random-value parent-sampling-constant
```

Comment:
```suggestion
sampling-constant      = sampling-random-value parent-sampling-rate
```

Reply:
It is not really the rate. It is a constant used to calculate the probability. For example, a value of 2 works out to a probability of `p = 2^-x = 2^-2 = 0.25`

Reply:
the term "parent sampling constant" has no meaning to me. It is still a sampling rate, it's just encoded in a particular manner.

Reply:
The way we have it specified in https://github.com/open-telemetry/oteps/pull/168, this field contains the "adjusted count" specifically encoded by the logarithm for non-zero values and as 63 for the zero case.